services/port_manager/src/main/java/com/futurewei/alcor/portmanager/repo/PortRepository.java

```diff
@@ -243,4 +257,67 @@ public PortNeighbors getPortNeighbors(Object arg) throws CacheException {
 
         return portNeighbors;
     }
+
+    private String getNeighborCacheName(String suffix) {
```

Comment:
@chenpiaoping  Piaoping, our debug session doesn't include the commit 27d5b64 "Change the name of neighbor cache" which was just committed right before the meeting. Do you think this would make the difference?

Also can you pull from the latest master later today as we want to make sure your private branch has all the master payload? Thanks.

Reply:
This commit will not affect the results of the test. I pulled it from master yesterday.